samcli/commands/list/resources/cli.py

```diff
@@ -0,0 +1,43 @@
+"""
+Sets up the cli for resources
+"""
+
+import click
+
+from samcli.cli.main import pass_context
+
+HELP_TEXT = """
+Get a list of resources that will be deployed to CloudFormation.\n
+If a stack name is provided, the corresponding physical IDs of each
+resource will be mapped to the logical ID of each resource.
+"""
+
+# @click.command(name="resources", cls=GenerateEventCommand, help=HELP_TEXT)
+
+
+@click.command(name="resources", help=HELP_TEXT)
+@click.option(
```

Comment:
We should probably also include these decorators common to most commands:
`@common_options`, `@aws_creds_options`, `@check_newer_version` and `@print_cmdline_args`.

Reply:
added those in